### PR TITLE
Remove taxonomies from config

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

Remove old taxonomy key and return all corpora in config endpoint for superusers
[PDCT-1052](https://linear.app/climate-policy-radar/issue/PDCT-1052/modify-api-config-endpoint)

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/repository/config.py [hunk 1]
```diff
@@ -89,24 +90,45 @@ def _to_corpus_data(row, event_types) -> CorpusData:
...
-            Corpus.import_id.label("corpus_import_id"),
-            Corpus.title.label("title"),
-            Corpus.description.label("description"),
-            Corpus.corpus_type_name.label("corpus_type"),
-            CorpusType.description.label("corpus_type_description"),
-            CorpusType.valid_metadata.label("taxonomy"),
+def get_corpora(
+    db: Session, user_email: str, is_superuser: bool
+) -> Sequence[CorpusData]:
+    if is_superuser:
```
Comment (diversemix) [inline app/repository/config.py:96]: DRY: can we not pull out the main common part of the query from the if/else clause?

**Review by diversemix — DISMISSED** [review]
Looks good :+1:

File: app/repository/config.py (hunk 1 above)
Comment (katybaulch) [reply app/repository/config.py]: Done

**Review by diversemix — APPROVED** [review]
lgtm

**Pull request merged**